dpnp/backend/backend_iface.hpp

```diff
@@ -400,6 +400,10 @@ INP_DLLEXPORT void custom_var_c(
     template <typename _DataType_input, typename _DataType_output>                                                     \
     INP_DLLEXPORT void custom_elemwise_##__name__##_c(void* array1, void* result1, size_t size);
 
+#define MACRO_CUSTOM_1ARG_2TYPES_2OPS(__name__, __operation1__, __operation2__)                                        \
```

Comment:
Why we need extra macro? It is the same as existed `MACRO_CUSTOM_1ARG_2TYPES_OP`

Reply:
Done!